From 26124dbb1960ef5b92bd3001cc71bf3398139b2f Mon Sep 17 00:00:00 2001
From: wagerrtor <tor@i2pmail.org>
Date: Tue, 14 Aug 2018 18:54:01 +0200
Subject: [PATCH] Replace SNAP's home dir

---
 src/util.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/util.cpp b/src/util.cpp
index c714b36..5405746 100644
--- a/src/util.cpp
+++ b/src/util.cpp
@@ -434,7 +434,7 @@ boost::filesystem::path GetDefaultDataDir()
     return GetSpecialFolderPath(CSIDL_APPDATA) / "WAGERR";
 #else
     fs::path pathRet;
-    char* pszHome = getenv("HOME");
+    char* pszHome = getenv("SNAP_USER_COMMON");
     if (pszHome == NULL || strlen(pszHome) == 0)
         pathRet = fs::path("/");
     else
-- 
2.17.1

