From ec230421e7107448ca9ac2421eb702924e12a879 Mon Sep 17 00:00:00 2001
From: observerdev <dev@obsr.org>
Date: Thu, 11 Apr 2019 00:12:00 +0200
Subject: [PATCH] fix-use-snap-instead--of-dirty

---
 share/genbuild.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/share/genbuild.sh b/share/genbuild.sh
index 519cc6e..de56b8d 100755
--- a/share/genbuild.sh
+++ b/share/genbuild.sh
@@ -37,7 +37,7 @@ if [ "${BITCOIN_GENBUILD_NO_GIT}" != "1" -a -e "$(which git 2>/dev/null)" -a "$(
 
     # otherwise generate suffix from git, i.e. string like "59887e8-dirty"
     SUFFIX=$(git rev-parse --short HEAD)
-    git diff-index --quiet HEAD -- || SUFFIX="$SUFFIX-dirty"
+    git diff-index --quiet HEAD -- || SUFFIX="$SUFFIX-snap"
 
     # get a string like "2012-04-10 16:27:19 +0200"
     LAST_COMMIT_DATE="$(git log -n 1 --format="%ci")"
-- 
2.17.1

